gcb/makestage/cloudbuild.yaml

```diff
@@ -13,15 +21,15 @@ steps:
     git clone "${_CM_REPO}" . && git checkout "${_CM_REF}"
 
 ## Build release artifacts and push to a bucket
-- name: 'eu.gcr.io/jetstack-build-infra-images/bazelbuild:${_BUILDER_IMAGE_TAG}'
+- name: 'europe-west1-docker.pkg.dev/cert-manager-tests-trusted/cert-manager-infra-images/make-dind:${_BUILDER_IMAGE_TAG}'
```

Comment:
To match cert-manager:
 * https://github.com/cert-manager/cert-manager/pull/7199
 * https://github.com/cert-manager/cert-manager/blob/f3b2a984482b36558b65a1511baffdccf38af616/gcb/build_cert_manager.yaml#L17-L19